### ISSUE TITLE
Feature request: Use PHP with same extension availability as PlatformSH

### ISSUE THREAD

**Issue by ptmkenny** [ddev/ddev-platformsh#128]
PlatformSH has a nice chart of all the PHP extensions they support ([default and available if enabled in php.ini](https://docs.platform.sh/languages/php/extensions.html)).

It would be super awesome if using this addon would automatically make such a PHP installation available.

I ran into this trying to enable `yaml`, which PlatformSH includes but ddev does not.

For those who need this now, you can add your own debian packages to provide the extensions you need:
https://ddev.readthedocs.io/en/stable/users/extend/customizing-images/#adding-extra-debian-packages-with-webimage_extra_packages-and-dbimage_extra_packages

**Comment by rfay** [comment]
Agreed it would be great to match up the extensions. DDEV has so many extensions by default I'm surprised you didn't have php*-yaml.

An issue to add php*-yaml to ddev would be welcome over there as well, if you think it's used often enough to be default.
